libraries/botbuilder-ai/src/languageTranslator.ts

```diff
@@ -29,20 +31,33 @@ export interface TranslationContext {
  * get a translated experience, and also a LUIS model allowing the user to ask to speak a language.
  */
 export class LanguageTranslator implements Middleware {
```

Comment:
>Middleware [](start = 43, length = 10)

This is going to need to be updated to use the M2 version of middleware.

Reply:
Is the M2 version of the middleware merged on the master branch yet?